tests/integration/integration_suite_test.go

```diff
@@ -1,4 +1,4 @@
-package integration_test
+package integration
```

Comment:
Why?

Reply:
I was matching the directory name. It makes it easier IMO to understand where packages live by just checking the directories layout.

Reply:
The _test has a meaning though in go's packages (please read it). It still has to follow directory / directory_test structure.

Reply:
Just read about this, thanks for the pointer.
But it doesn't seem to apply here as we're not really testing an `integration` packages (which would require then to use `integration_test`).
I've checked the package for the e2e tests of Kubernetes and [it's called e2e](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/e2e.go#L17), not e2e_test.

Am I misunderstanding your point here? What would we get by naming the package `integration_test`?

Reply:
I wanted you to understand the change before making them, instead of making changes without understanding the effect. We have no need for _test in this case.